packages/server/src/adapters/lambda-utils.ts

```diff
@@ -0,0 +1,30 @@
+import type { APIGatewayProxyEvent } from 'aws-lambda';
```

Comment:
To be consistent with the other adapters, let's create a separate directory for this and put `aws-lambda` and `lambda-utils` inside it. As a side note, tRPC uses a camelCase file naming convention. So the renaming should be like this:
```ts
adapters/aws-lambda.ts -> adapters/lambda/lambdaRequestHandler.ts
adapters/lambda-utils.ts -> adapters/lambda/types.ts
```